translations/ui-agreements/en.json

```diff
@@ -569,10 +569,10 @@
   "settings.hideEResources.result.3": "Hide the \"E-resources covered by this agreement\" accordion in individual agreements",
   "settings.general.displaySuppressFromDiscovery.description": "Display \"Suppress from discovery\" setting on",
   "settings.general.displaySuppressFromDiscoveryAgreementLine.title": "Agreement lines",
-  "settings.general.displaySuppressFromDiscoveryPCI.title": "Package content items",
-  "settings.general.displaySuppressFromDiscoveryTitle.title": "Title instances",
-  "settings.mcl.pagination.description": "<strong>MCL Pagination</strong>",
-  "settings.mcl": "<strong>MCL</strong>",
+  "settings.general.displaySuppressFromDiscoveryPCI.title": "Titles in packages",
```

Comment:
I would suggest we change the `key` here to be more specific such as 
```settings.general.displaySuppressFromDiscovery.titlesInPackages```

and change its corresponding usage in the code as well